frameworks/C++/drogon/drogon_benchmark/plugins/SyncPlugin.cc

```diff
@@ -0,0 +1,51 @@
+/**
+ *
+ *  SyncPlugin.cc
+ *
+ */
+
+#include "SyncPlugin.h"
+#include <drogon/drogon.h>
+
+using namespace drogon;
+
+void SyncPlugin::initAndStart(const Json::Value &config)
+{
+    /// Initialize and start the plugin
+    drogon::app().registerSyncAdvice(
+        [](const HttpRequestPtr &req) -> HttpResponsePtr {
+            if (req->method() != Get)
+            {
+                return HttpResponsePtr{};
+            }
+            switch (req->path().length())
+            {
+                case 5:
+                    if (req->path() == "/json")
```

Comment:
Curious why you're checking the path string if you're already switching over the length? Why not one or the other?

Reply:
Consider the `/db` path of the single query test. When processing a request with this path, if you use a switch statement like this, there is no comparison of the string, otherwise there will be two string comparisons. I think this has a little benefit on performance.
If in the actual environment, many requests have the same prefix or there are more valid path strings to compare, I believe this will bring more benefits.
And if an invalid request with a length of 5 is processed here, it is necessary to perform a string check after switching over the length.